test/pg-cdc/pg-cdc.td

```diff
@@ -101,15 +101,15 @@ incorrect column specification: specified column does not match upstream source,
     PUBLICATION 'mz_source_${testdrive.seed}'
     NAMESPACE 'public'
     TABLES ('numbers_${testdrive.seed}' AS "numbers")
-CREATE SOURCES not yet supported
+Postgres sources not yet supported
```

Comment:
Nit: Keeping these error messages in the exact same case the `str` that generated them in the code base makes things easier to find. Looks like this is [`"Postgres Sources"`](https://github.com/MaterializeInc/materialize/pull/6448/files#diff-c840a7fbb6a642d6cbf71dddd92ec1b6dfa86310022ead5673e8fa509ba73dc4R1042) above.